convert2rhel/unit_tests/main_test.py

```diff
@@ -123,6 +123,7 @@ def test_rollback_changes(self):
 
 
     @unit_tests.mock(main.logging, "getLogger", GetLoggerMocked())
+    @unit_tests.mock(main.special_cases, "check_and_resolve", lambda: None)
```

Comment:
Do not allow to run this

Reply:
Should be using `CallOrderMocked` here to check that the function gets called, same on L163

Reply:
ok, will do